### PR TITLE
Re-enable automatic deployments in gitlab

### PR DESCRIPTION
**Pull request by kislyuk**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -38,6 +39,8 @@ deploy:
       - master
       - integration
       - staging
+  variables:
+    TF_CLI_ARGS_apply: -auto-approve
```
Comment (MDunitz) [inline .gitlab-ci.yml]: allowing gitlab to autoapprove terraform changes seems a little dangerous. Is there a reason to move away from the old process of running make plan and letting the build fail if it returns changes so those can be applied manually?

Comment (kislyuk) [reply .gitlab-ci.yml]: This is a great point, I think I'll rework this to do it the way you expected.

Comment (kislyuk) [reply .gitlab-ci.yml]: Upon further consideration, I would like to make deployments in the dev account (dev, integration, and staging environments) fully automatic. The prod deployment will need to be manual. To mitigate possible risk from this, we can look at deletion protection for database resources.

**Comment by kislyuk** [comment]
Merged in https://github.com/HumanCellAtlas/query-service/pull/155.

**Pull request closed without merging**